app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="auto_size_text_sizes">
```

Comment:
You have not used it anywhere right? You can remove whole file I think.

Reply:
yep removed